Jenkinsfile

```diff
@@ -36,7 +36,6 @@ spec:
           strategy: ephemeral
   transports:
     qdr:
-      auth: none
```

Comment:
🥳

Reply:
There is an equivalent change pending in stf-v-c MR#103

Reply:
🥳++